__tests__/__unit__/job/actions.unit.test.ts

```diff
@@ -60,17 +60,17 @@ describe("Jobs Actions Unit Tests - Function setPrefix", () => {
     let blockMocks;
```

Comment:
The same comment I gave to Stepan yesterday. This should become `const` and to into individual `it` blocks. But we will tackle this in all files at once after we merge the current slate of PRs.

Reply:
I can make this change in the last PR